### DISCUSSION TITLE
May i make us a patreon for supporters

### DISCUSSION THREAD

**Discussion by irhrhd** [Prodigy-Hacking/ProdigyMathGameHacking#662] [(A) Anything]
Patreon

**Comment by BoredFishRE** [comment]
Hell no

**Comment by BoredFishRE** [comment]
A. We don't take money. B. Why should we trust you with our money.

**Comment by irhrhd** [comment]
Oh i mean give you guys the account so you guys dont have to always pay out
of your pocket and u guys can change the password

**Comment by BoredFishRE** [comment]
Then why wouldn't we just create the acc

**Reply by ghost** [reply]
i mean like you guys should create some kind of donation thing (btw gofundme allows people under 18, just ask your parents for a bank account) but no need to let this kid create it.

like he's trying to help the repo but his attempts are quite misguided.

**Reply by PatheticMustan** [reply]
hmm, I think if one of us could set up a github sponsor button, we might be able to have it pay for some of the VPS costs

**Reply by BoredFishRE** [reply]
the problem is that one person would have access, and it would be under one person, we can't make like a group bank account.

**Reply by ghost** [reply]
> the problem is that one person would have access, and it would be under
> one person, we can't make like a group bank account.

i mean you could ask your parents for a bank account then give the other people access or smth.

i dont have a bank account so idk much about this but i'll try to think about it

**Reply by BoredFishRE** [reply]
I'm not 100% sure that's legal.

**Reply by ghost** [reply]
online banks probably have a way to legally share your account

maybe natsumi could open the bank account because he's the owner

**Comment by irhrhd** [comment]
Thats what i was thinking

**Comment by BoredFishRE** [comment]
If anything @irhrhd, we would make it.

**Comment by irhrhd** [comment]
Okay thanks for the feedback bored
